test/controllers/api-key.test.js

```diff
@@ -24,60 +34,167 @@ describe('ApiKeyController tests', () => {
   let context;
   let apiKeyController;
   let requestContext = {};
+  let exampleApiKey;
 
   beforeEach(() => {
     requestContext = {
       pathInfo: {
         headers: {
-          'x-gw-ims-org-id': 'test-org',
```

Comment:
Looking at trials and ALPS, I think that `x-gw-ims-org-id` is actually the correct header name to use.

Reply:
Done.